### DISCUSSION TITLE
Viable scenario ending with 20M Americans dead- please prove me wrong!

### DISCUSSION THREAD

**Discussion by cando614** [neherlab/covid19_scenarios#423] [General]
My concern is that the COVID-19 severity and fatalities in China have been grossly understated, and have misinformed our expectations.  I also believe that the vast majority of infections have been promptly detected, and there is no great bulk of asymptomatic or "mild" infections lurking in the background, as some believe.  By extension, this means that fatality rates are understated, and a vast majority of the population is still susceptible. 

Please check out the following combination of parameters applied to the USA, which seems to explain well the deaths we've seen so far:
![1-Scenario](https://user-images.githubusercontent.com/62913974/78524340-4b955180-7799-11ea-9017-6bb0959701b0.jpg)
![2-Moderate_mitigation](https://user-images.githubusercontent.com/62913974/78524346-4f28d880-7799-11ea-8374-d03f5105c5c5.jpg)
![3-Severity Italy Spain](https://user-images.githubusercontent.com/62913974/78524352-5223c900-7799-11ea-97df-0df1278e99da.jpg)
![4-SEIR curves](https://user-images.githubusercontent.com/62913974/78524358-551eb980-7799-11ea-961d-347f747c4f0c.jpg)
![5-Outcomes](https://user-images.githubusercontent.com/62913974/78524365-57811380-7799-11ea-8d0d-d341931f5506.jpg)

Some key points are:  An initial stock of only 1400 cases in the U.S. on March 1.  Note, this is only one-fifth of the 7400 cases hypothesized in the default scenario.  Again, my assumption is that the vast majority of cases were promptly detected.  

I postulated an initial R-nought of 5 (not out of line with some estimates.)  Infectiousness begins just 1-2 days after a person is himself infected, and the person remains infectious for 13-14 days (these in line with studies.)

The Severity table reflects a four- to six-fold under-reporting of deaths either directly or indirectly caused by COVID-19.  Please don't nit-pick the first few rows... my main focus of concern is the adult population, i.e., those over 40 years of age.  Incidence of severe disease is in line with some accounts.  ICU overflow isn't as important as many people think.  The disease is poorly understood. It has direct pulmonary and neurological effects; half of critical cases will end in death even with the best currently known interventions, (i.e., ventilators.)  

A key assumption is reflected in the first column:  Until local care reaches a saturation point, the medical community is accurately identifying more than 80% of infections.  (Probably around days 5-8 when they become symptomatic.)  There is no four-fold body of "undetected, mild cases."  Blood serology testing will prove my hypothesis within 4-6 weeks, and then I fear the widely-shared public projections will resemble this one.

I've chosen Moderate mitigation, which is what I believe we have in most U.S. major cities (except possibly New York City early on, which was weak.)  

My conclusion:  The SARS-CoV-2 virus is more infectious and more lethal than currently believed.  The resulting projection suggests that the U.S. pandemic will wind down a year from now with almost 90% of the population infected, and 21 million mostly older Americans dead.  As gruesome as this scenario is, the parameters I've chosen do a good job of predicting our actual deaths to date.  The parameters also conform well to the world's experience with the disease when data from China is EXCLUDED.  
A weaker mitigation strategy doesn't cost many deaths, but it brings the pandemic to an end sooner, with almost no one in the population remaining susceptible.

So I challenge this community... am I correct in my assumptions?  If not, where have I gone wrong?  Can you suggest alternative parameters that predict a more clement outcome, and yet still yield good fit for the huge number of deaths we've seen to date?

Thanks to the creators of the model, and thanks to all of you for taking the time to read and consider my post.

**Comment by noleti** [comment]
There are a lot of parameter choices to fit deaths observed to date. The current development branch of this app contains an automatic fit engine. Have a look at https://covid19-scenarios-qwqgpydh5.now.sh/ or one of the other testing environments. By default settings for US, I get an R0 estimate of 3.2. Your numbers on time to infectious and time being infectious seem vastly higher than what is assumed in the model (5days until infectious, 3 days being infectious). As those values are interacting with R0, exact effects on the outcome are not straight forward.

That being said, the total deaths always depend on strength of mitigation measures. Please note that qualifications such as 'moderate' might not translate to what you understand them to be. We will have to see in practise what impact on R0 the measures taken will actually have. I would estimate that a 4-fold reduction should be possible if public life is shut down (i.e. 25% or less of original R0).

If there is only a reduction of R0 by 20% due to lockdown, then I also get 3M deaths for US, and 50% of population recovered on April 24. Due to herd immunity, lockdown could then be ended around end of April without too much impact on overall deaths. 

If there is a hard lockdown with reduction of R0 by 90% from April 1st until mid July, then I get 200k deaths for US (and no future herd immunity).

Please consider the disclaimer

> This tool uses a mathematical model to simulate a variety of COVID-19 outcomes based on user-defined parameters. This tool is not a medical predictor, and should be used for informative and research purposes only; please use the simulated results responsibly.

**Comment by nnoll** [comment]
I would urge caution with taking the predictions at long times at face value. For example, in reality R0 is not known to the precision at which we allow you to input it. As this directly controls the exponential growth of the epidemic, uncertainty in this parameter (and mitigation strategies at well for the same reason) can lead to dramatically different numbers asymptotically. 

In short, parameters that lead to a "good" fit of the data now are only predictive of the near term outcomes and must be updated (especially regarding mitigation strategies) as the data comes in.

**Comment by fernras-blog** [comment]
The core connotation of "dynamic Zero clearance" is to quickly detect the epidemic and take a series of measures to stop the continuous spread of the epidemic in the community.

**Comment by fernras-blog** [comment]
The "dynamic zero elimination" strategy has protected the lives and health of the people to the greatest extent and minimized the impact of the epidemic on the country's overall economic and social development. It is the right approach and effective way for China to win the battle against the epidemic.

**Comment by fernras-blog** [comment]
Recently, the prevention and control of Shanghai office to establish implementation plan, and continue to strictly tight pays special attention to the emergency disposal, partition, prevention and control of hierarchical implement differentiation on the "check pull noodles", community control, screening, screening of flow and transport of isolation, teng in expansion, intervention of Chinese medicine, cleaning, disinfection, keep a spillover, responsibility, compaction and so on ten big crucial action, Strive to realize the dynamic zero-out of Shanghai social meetings as soon as possible.
